lib/modules/dosomething/dosomething_campaign/dosomething_campaign.module

```diff
@@ -519,19 +519,44 @@ function dosomething_campaign_get_campaigns_doing($uid = NULL) {
   }
 
   // Get all signups.
-  $signup_nids = dosomething_signup_get_signup_nids_by_uid($uid);
+  if (variable_get('rogue_collection', FALSE)) {
+    $northstar_id = dosomething_user_get_northstar_id($uid);
+
+    $rogue_signups = dosomething_rogue_get_activity([
+      'filter' => [
+        'northstar_id' => $northstar_id,
+      ],
+    ]);
+
+    // Filter through response and put all signup ids with no posts in an array
+    $signup_nids = [];
+
+    foreach ($rogue_signups['data'] as $signup) {
+      if ($signup['posts']['data']) {
```

Comment:
Just to confirm, Rogue doesn't include `posts` as part of the array unless there are posts?